lib/ex_doc/markdown.ex

```diff
@@ -8,9 +8,9 @@ defmodule ExDoc.Markdown do
   * [Earmark][]
   * [Pandoc][]
 
-  If you don't offer any preference via `config/config.exs`. ExDoc will try to
-  find one of the earlier MarkDown parsers. Otherwise, ExDoc will raise an
-  exception.
+  If you don't specify a parser in `config/config.exs`. ExDoc will try to
+  find one of the MarkDown parser from list in top down fashion. Otherwise, 
```

Comment:
Hey Eric,
you are right it should read as
"ExDoc will try to find one of the MarkDown parser from the list in top down fashion"

Thanks